src/main/resources/build.properties

```diff
@@ -6,3 +6,5 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Single instance=????
```

Comment:
The Problem is that IntelliJ suggests the wrong file:
You need to put these strings into  src/main/resources/l10n/en.properties

Reply:
https://devdocs.jabref.org/code-howtos/localization.html

Reply:
> https://devdocs.jabref.org/code-howtos/localization.html

Should I modify it in GeneralTab.class?